src/overlays/actors/ovl_En_Go/z_en_go.c

```diff
@@ -1815,137 +2357,160 @@ void func_80A153FC(EnGo* this, PlayState* play) {
     }
 }
 
-s32* func_80A15684(EnGo* this, PlayState* play) {
-    static s32 D_80A16704[] = {
-        D_80A16100,
-        D_80A16164,
+/**
+ * Return the message script appropriate for the actor
+ *
+ * @returns Message Script corresponding to the actor
```

Comment:
Message usually means the text system, `z_message`, so better to say `MsgEvent` so it gets caught when we decide how better to name that system.